tests/test_compiler.py

```diff
@@ -43,7 +43,7 @@ def catalyst_incompatible_version():
 
 @pytest.mark.usefixtures("catalyst_incompatible_version")
 def test_catalyst_incompatible():
-    """Test qjit with an incompatible Catalyst version < 0.4.0"""
+    """Test qjit with an incompatible Catalyst version < 0.6.0"""
```

Comment:
We should generalize this so it doesn't need updating every time
```suggestion
    """Test qjit with an incompatible Catalyst version that's lower than required."""
```